src/layout/section/itemsSection/itemsSection.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+
+import { ItemInfoProps } from 'itemInfo'
+
+import BaseSection from 'layout/section/baseSection'
+
+export interface ItemsSectionProps {
+  readonly children: React.ReactElement<ItemInfoProps>[]
+  readonly className?: Classcat.Class
+  readonly tag?: JSX.Element
+}
+
+/**
+ * Items Section: display a list of items in a display: flex.
+ * Use with two items.
+ */
+const ItemsSection = (props: ItemsSectionProps) => {
+  const { className, children, tag = <div /> } = props
+
+  return (
+    <BaseSection tagName={tag.type} contentClassName={className}>
```

Comment:
className param should go into className of the BaseSection.
Styled-components will automagically inject a class for its styling and classname should be on the root (mostly for consistency with the rest of our widgets).

You need something like:

```
const ItemsSection = (props: ItemsSectionProps) => {
  const { className, children, tag = <div /> } = props

  return (
    <BaseSection tagName={tag.type} className={className} contentClassName='kirk-items-section-content'>
      {children}
    </BaseSection>
  )
}
```

And in your index.tsx:
```
& .kirk-items-section-content {
  display: flex;
}

```